docs/Import-reference-genome.md

```diff
@@ -0,0 +1,47 @@
+# Add new reference genome to the database
+This manual is intended to guide you to load new reference genome(s) to the database. 
```

Comment:
@sandertan, because we display **cytoband** in the CNA table which is tied with genomic coordinates. Also, those plots on the patient view are using genomic coordinates.

Reply:
also added more details in Import-reference-genome.md file.